src/tsickle.ts

```diff
@@ -758,6 +758,7 @@ class Annotator extends ClosureRewriter {
         }
 
         this.emitFunctionType([fnDecl], tags);
+        this.newTypeTranslator(fnDecl).blacklistTypeParameters(fnDecl.typeParameters);
```

Comment:
How does this work?  It looks like it creates a type translator, modifies its state, then throws it away...?

Reply:
I've tried to document on the method we're calling. Indeed, this mutates the Map that we're using for aliased symbols. Not super cool, but OTOH pretty harmless, and keeps the blacklisting code in one place. Any other ideas?

Reply:
But it only mutates the map on the TypeTranslator instance, so how does the change persist?

Reply:
It's the same map - we're using only one map of aliases per file.

Reply:
I've changed the method to take the map as a parameter, and added more docs.